components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java

```diff
@@ -492,19 +488,31 @@ public static void advUsage() {
     private static List<Annotation> toTextAnnotations(
                    List<String> namespaces, List<String> strings)
     {
-        if (namespaces.size() != strings.size())
-        {
-            throw new IllegalArgumentException(String.format(
-                            "#Namespaces:%d != #Text:%d", namespaces.size(),
-                            strings.size()));
-        }
         List<Annotation> annotations = new ArrayList<Annotation>();
-        for(int i = 0; i < namespaces.size(); i++)
+        CommentAnnotationI annotation;
+        if (namespaces.size() != strings.size())
```

Comment:
I guess you did not use my branch or the merge branch locally
that's why you see the issue
eel-merge user-1:

``bin/omero import -d 18854 --annotation-text testAnnotation1  ~/Documents/1.ome.tiff ``
the image is imported and has one text annotation

Could you try again?